docs/tutorials/otio-serialized-schema.md

```diff
@@ -361,13 +361,52 @@ parameters:
 *documentation*:
 
 ```
-Refers to a sequence of image files to be played back at a constant rate.
+Refers to a sequence of image files to be played back
+                    at a constant rate.
+
+                    The target urls for the images are determined by a concatentation of
+                    ``target_url_base``, ``name_prefix``, frame number (applying
+                    ``frame_zero_padding``) and ``name_suffix``.
+                    For instance, if the following values are used::
+                        'target_url_base': 'file:///show/sequence/shot/',
+                        'name_prefix': 'sample_image_sequence.',
+                        'name_suffix': '.exr'
+                        'frame_zero_padding': 4
+
+                    Then the target url for frame 7 in the sequence would be::
+                        file:///show/sequence/shot/sample_image_sequence.0007.exr
+
```

Comment:
The `ImageSequenceReference` doesn't have the standard `target_url()` method does it? I'm afraid referring to "target url" will mislead people into calling that method. Perhaps including the call for the url in the example will avoid confusion?
``` shell
>>> my_mediareference.target_url_for_image_number(7)
>>> 'file:///show/sequence/shot/sample_image_sequence.0007.exr'
```